packages/contracts/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "0x-smart-contracts",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "transpile": "rm -rf ./transpiled; copyfiles ./build/**/* ./deploy/solc/solc_bin/* ./deploy/test/fixtures/contracts/**/* ./deploy/test/fixtures/contracts/* ./transpiled; tsc;",
+    "test": "npm run transpile; truffle test",
+    "compile": "npm run transpile; node transpiled/deploy/cli.js compile",
+    "migrate": "npm run transpile; truffle migrate",
+    "migrateCustom": "npm run transpile; node transpiled/deploy/cli.js migrate",
+    "deploy": "npm run migrate --network kovan",
+    "lint": "tslint --project . 'migrations/*.ts' 'test/**/*.ts' 'util/*.ts' 'deploy/**/*.ts'",
+    "testrpc": "testrpc --networkId 50",
+    "mocha": "npm run transpile; mocha transpiled/deploy/test/*_test.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/0xProject/0x-Smart-Contracts.git"
+  },
+  "author": "abandeali1@gmail.com",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/0xProject/0x-Smart-Contracts/issues"
+  },
+  "homepage": "https://github.com/0xProject/0x-Smart-Contracts#readme",
+  "devDependencies": {
+    "@types/bluebird": "^3.5.3",
+    "@types/isomorphic-fetch": "^0.0.34",
+    "@types/lodash": "^4.14.63",
+    "@types/node": "^8.0.51",
+    "@types/request-promise-native": "^1.0.2",
+    "@types/yargs": "^8.0.2",
+    "chai": "^4.1.2",
+    "chai-as-promised": "^7.1.1",
+    "chai-as-promised-typescript-typings": "^0.0.3",
+    "chai-bignumber": "^2.0.1",
+    "chai-typescript-typings": "^0.0.0",
+    "copyfiles": "^1.2.0",
+    "dirty-chai": "^2.0.1",
+    "ethereumjs-testrpc": "4.0.1",
+    "mocha": "^4.0.1",
+    "solc": "^0.4.18",
+    "truffle": "3.4.3",
+    "tslint": "5.8.0",
+    "tslint-config-0xproject": "^0.0.2",
+    "types-bn": "^0.0.1",
+    "types-ethereumjs-util": "machinomy/types-ethereumjs-util",
```

Comment:
Why are we using this fork? We use `0xProject/types-ethereumjs-util` in 0x.js. It's safer to use our own fork because `machinomy` might decide to remove his repo in the future for the lols.

Reply:
We use our fork in 0x.js because of that https://github.com/machinomy/types-ethereumjs-util/pull/1.
It's merged but not released yet. I'll ping him